src/static/styles/main.scss

```diff
@@ -19,31 +18,28 @@ body {
 }
 
 body {
-    font-family: Roboto, Arial, Helvetica, sans-serif;
-    font-size: 16px;
     line-height: 1.2em;
-    color: var(--primary);
     margin: 0;
-    background-color: var(--background);
 }
 
-.title-bar {
-    padding: 24px;
-    position: relative;
-    text-align: center;
-    font-weight: 500;
-
-    .go-back {
-        position: absolute;
-        left: 24px;
-        top: 24px;
+input,
+button {
+    font-family: var(--font-main);
```

Comment:
I see that the text in (text) input fields is tiny compared to what it was before, at least in Firefox. Is this the culprit?

Reply:
Inputs should be explicitly set as 16px here I think. This is also to prevent iOS to zoom in on focus.

Reply:
It says 11px in computed in my Firefox, but I see it tries to use `1rem` in the rules tab.

Reply:
Also. I've removed some of the default styles for button/inputs which cause some issues elsewhere. But going forward using UI Components this will be resolved.

Reply:
> It says 11px in computed in my Firefox, but I see it tries to use 1rem in the rules tab.

Yep, but I pushed explicit 16px now, which should fix it for now. Mainly this is for iOS phone zoom though. The design is going to get somewhat in flux until we refactor code to use UI components. I'm gradually removing default styling on things to avoid having to reset it and to be more interoperable/pluggable within different contexts